test/spmd/tensor/test_dtensor_ops.py

```diff
@@ -146,7 +146,7 @@ def wrapped(fn):
     xfail("_masked.norm"),
     xfail("_masked.prod"),
     xfail("_masked.softmin"),
-    xfail("_masked.softmax"),
+    #xfail("_masked.softmax"),
```

Comment:
Instead of commenting it out, can we remove it directly?

Reply:
Will put this one back since it's the wrong test. `skip("softmax")` has been removed.